compiler/rustc_codegen_llvm/src/back/archive.rs

```diff
@@ -61,6 +64,15 @@ fn archive_config<'a>(sess: &'a Session, output: &Path, input: Option<&Path>) ->
     }
 }
 
+/// Map machine type strings to values of LLVM's MachineTypes enum.
+fn llvm_machine_type(cpu: &str) -> LLVMMachineType {
+    match cpu {
+        "x86_64" => LLVMMachineType::AMD64,
+        "x86" => LLVMMachineType::I386,
+        _ => panic!("unsupported cpu type {}", cpu),
```

Comment:
Might as well add the arm/aarch64 cpu types here too since nothing else in the PR is x86* specific.